native/src/components/PrimaryTextButton.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { ReactElement } from 'react'
+import { Button } from 'react-native-elements'
+import { IconNode } from 'react-native-elements/dist/icons/Icon'
+import styled, { useTheme } from 'styled-components/native'
+
+const Container = styled.View<{ $padding: boolean }>`
+  padding: ${props => (props.$padding ? '16px' : '0')};
+`
+
+type PrimaryTextButtonProps = {
+  onPress: () => void
+  text: string
+  disabled?: boolean
+  Icon?: IconNode
+  padding?: boolean
+}
+
+const PrimaryTextButton = ({
+  onPress,
+  text,
+  padding = true,
+  disabled = false,
+  Icon,
+}: PrimaryTextButtonProps): ReactElement => {
+  const theme = useTheme()
+
+  return (
+    <Container $padding={padding}>
+      <Button
+        icon={Icon}
+        onPress={onPress}
+        title={text}
+        disabled={disabled}
+        titleStyle={{
+          color: theme.colors.textColor,
+          fontFamily: theme.fonts.native.contentFontBold,
+          fontSize: 14,
+        }}
+        buttonStyle={{
+          backgroundColor: theme.colors.themeColor,
+          borderRadius: 8,
```

Comment:
This changes the borderRadius for some buttons compared to before. Although I see that you probably tried to align this with the new feedback design right?

Reply:
Yes, both the new feedback design and the design for the poi filter (https://www.figma.com/file/cA4F2MwHs2LNGviOWUjfE9/Integreat-Tickets-Jira?node-id=961%3A3409&t=HoTJnEmjdDKZVa3n-1) have the rounded buttons and I figured it makes sense to make it consistent. You think we shouldn't?

Reply:
No, it is fine as it is. Although, I personally prefer the corners. :)